src/NServiceBus.Core/UnitOfWork/UnitOfWorkBehavior.cs

```diff
@@ -42,7 +47,7 @@ class UnitOfWorkBehavior : Behavior<PhysicalMessageProcessingContext>
             }
         }
 
-        List<Exception> AppendEndExceptionsAndRethrow(Exception initialException)
+        List<Exception> AppendEndExceptionsAndRethrow(Stack<IManageUnitsOfWork> unitsOfWork, Exception initialException)
```

Comment:
I fixed it better:
![image](https://cloud.githubusercontent.com/assets/601206/11392519/017413d0-9363-11e5-8235-a7db53147da0.png)